src/app/public/modules/timepicker/timepicker.component.ts

```diff
@@ -114,12 +114,12 @@ export class SkyTimepickerComponent implements OnInit {
 
   public get selectedTime() {
     const time: SkyTimepickerTimeOutput = {
-      hour: moment(this.activeTime).hour(),
-      minute: moment(this.activeTime).minute(),
-      meridie: moment(this.activeTime).format('A'),
-      timezone: parseInt(moment(this.activeTime).format('Z'), 10),
+      hour: moment(this.activeTimeOrDefault()).hour(),
```

Comment:
seems like a fine approach, I'm just curious why not put the activeTimeOrDefault logic into an activeTime getter? Is there anywhere active time is used that it can't have a default?